docs/Dockerfile

```diff
@@ -15,11 +15,18 @@ RUN python3 -m venv $VIRTUAL_ENV
 ENV PATH="$VIRTUAL_ENV/bin:$PATH"
 RUN pip install --upgrade pip
 
+
+# ------------
+
+
 # Install ReactPy
 # ------------
 COPY requirements ./requirements
 RUN pip install -r requirements/build-docs.txt
```

Comment:
Given that you're installing the pinned versions of these dependencies below you can remove this.